src/database/guild.ts

```diff
@@ -0,0 +1,98 @@
+import { prisma } from "@database/client";
```

Comment:
no explicit return types in the entire file :whyyyy:

Reply:
Cause prisma already gives them, why the unnecessary effort

Reply:
makes it easier to read without lsp + you determine exactly what type you want returned from the function and then adjust the code to return that type. but again, this is fine